ci/appveyor/run-with-visual-studio.cmd

```diff
@@ -0,0 +1,88 @@
+:: To build extensions for 64 bit Python 3, we need to configure environment
```

Comment:
Was this final originally copied from https://github.com/cython/cython/blob/master/appveyor/run_with_env.cmd ? 

May be the commit message or the first line of the file could mention it.